src/Common/Authentication/Authentication.sln

```diff
@@ -0,0 +1,42 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2013
+VisualStudioVersion = 12.0.40629.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Common.Authentication", "Common.Authentication\Common.Authentication.csproj", "{B95C489C-8CB7-4DCF-8D5F-B9AEBDBBAF89}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Common.Authentication.Tests", "Common.Authentication.Tests\Common.Authentication.Tests.csproj", "{C2CF99A2-D35E-4AED-AFB9-C26960AF1D0D}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Net40-Debug|Any CPU = Net40-Debug|Any CPU
```

Comment:
There is no need for Net40 and Net45 build configurations (we should eventually remove them from AutoRest). In PowerShell it's just Debug and Release.